classes/warrior.lua

```diff
@@ -24,8 +24,8 @@ local function registerClass(self)
     self:RegisterCounter("revenge"); -- Must match name from above call
 
     -- Victory Rush
-    self:RegisterAura("victoryrush", 0, victoryrush, nil, "", 0, 0, 0, 0, false, { victoryrush });
-    self:RegisterCounter("victoryrush"); -- Must match name from above call
+    self:RegisterAura("victoryRush", 0, victoryRush, nil, "", 0, 0, 0, 0, false, { victoryRush });
+    self:RegisterCounter("victoryRush"); -- Must match name from above call

```

Comment:
The variable name in camelCase is good. But for string passed to RegisterAura and RegisterCounter: `"victoryRush"` 👉 `"victory_rush"`

Reply:
Done